### PR TITLE
bucket_name: add new variable bucket_name

### PR DESCRIPTION
**Pull request by gaelL**
This variable is here to be able to force the bucket name.

Remove the default behavior who name as site.$domain

### CODE REVIEW

**Comment by ringods** [comment]
Hello @gaelL,

Your contribution is much appreciated. However, to have the README.md in sync with the implementation, can you extend the docs (examples + variable description) to reflect your changes?

Tnx.

**Comment by gaelL** [comment]
Sorry it was expected to be an internal PR. If you are interested by this change, I will update the readme as well. Thx

**Comment by ringods** [comment]
@gaelL please do and I will integrate it in the repo here.

**Comment by gaelL** [comment]
done. I also add more information in the commit message

File: site-main/variables.tf [hunk 1]
```diff
@@ -3,6 +3,7 @@ variable region {
 }
 
 variable domain {}
+variable bucket_name {}
```
Comment (ringods) [inline site-main/variables.tf:6]: Can you make the module backwards compatible by setting the default value of the bucket_name to the calculated value it was before?

` "site.${replace("${var.domain}",".","-")}"`

**Review by ringods — CHANGES_REQUESTED** [review]

File: site-main/variables.tf (hunk 1 above)
Comment (gaelL) [reply site-main/variables.tf:6]: In fact no, I tried but terraform can't have interpolation in default variable 

https://github.com/hashicorp/terraform/issues/4084

**Pull request merged**